src/style.css

```
@@ -1,3 +1,82 @@
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
+:root {
+  --active-collor: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header-nav {
+  position: relative;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  background-color: #fff;
+}
+
+.nav-bar {
+  margin: 0 50px 0 0;
+}
+
+.nav-list {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav-list :first-child a {
+  margin: 0 10px 0 0;
+}
+
+.nav-list :last-child a {
+  margin: 0 0 0 10px;
+}
+
+.nav-link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 60px;
+  position: relative;
+  margin: 0 10px;
+  color: #000;
```

I see that you have already learned how to use CSS variables and actively use them in your code.
Always use the same style, avoid mixing(In all the cases)
```suggestion
  color: var(...)
```